app/screens/login_options/login_options.js

```diff
@@ -376,6 +376,7 @@ export default class LoginOptions extends PureComponent {
 const style = StyleSheet.create({
     container: {
         flex: 1,
+        backgroundColor: 'white',
```

Comment:
why are we setting the background color to white? shouldn't this screen follow the OS sync theme dark/light?